internal/pkg/manifest/nobl9/v1alpha/objects.go

```diff
@@ -0,0 +1,373 @@
+/*
```

Comment:
How about flattening the file structure from **internal/pkg/*** to **internal/*** ? The pkg directory within internal seems to be redundant.

Reply:
I thought about this, and i looked into it, and found [this](https://github.com/golang-standards/project-layout#internal) and that suggests using `internal/pkg` for app code that is shared. Thoughts?